Xamarin.Forms.Platform.Android/Renderers/MediaElementRenderer.cs

```diff
@@ -0,0 +1,310 @@
+using Android.Content;
+using Android.Media;
+using Android.Views;
+using Android.Widget;
+using System;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Android;
+using Android.Graphics;
+using Android.Runtime;
+
+namespace Xamarin.Forms.Platform.Android
+{
+	public sealed class MediaElementRenderer : ViewRenderer<MediaElement, FrameLayout>, MediaPlayer.IOnCompletionListener, MediaPlayer.IOnPreparedListener
```

Comment:
I am wondering if we need this one at all? What is the benefit of the non-fast renderer? We just add an extra view.